Source/Scene/Cesium3DTileset.js

```diff
@@ -1565,6 +1592,25 @@ define([
         // Number of commands added by each update above
         statistics.numberOfCommands = (commandList.length - numberOfInitialCommands);
 
+        if (addedCommandsLength > 0) {
+            // TODO : only do this if the tileset is purely point clouds.
+            // TODO : this may not work well if point cloud spans multiple frustums
+            // TODO : make the processor a static class so it can be used by multiple tilesets?
```

Comment:
Regarding the second comment, we currently have multifrustrum support in the code